.circleci/config.yml

```diff
@@ -90,6 +90,15 @@ jobs:
             source emsdk_env.sh
             python scripts/test.py
 
+  build-docker-image:
+    machine: true
+    steps:
+      - checkout
+      - run: docker build -t emscripten-core/emsdk:$CIRCLE_BRANCH docker/
```

Comment:
We do have docker.io organization called [emscripten/emsdk](https://hub.docker.com/repository/docker/emscripten/emsdk/general). 
I don't know who manages this organisation, looks like @haraldreingruber has access to another project [emscripten/emscripten-ci](https://hub.docker.com/repository/docker/emscripten/emscripten-ci/general) - maybe he knows more :)

Reply:
I seem to remember deciding on `emscripten/emsdk` for this image, right?  Or am I remembering wrong?

Reply:
Most likely you're right. And looking at the name of this repo it makes sense. 
In this case there should be a following action on https://hub.docker.com/orgs/emscripten

Reply:
The org (https://hub.docker.com/orgs/emscripten) was created by @kripken, right?
It's odd, but I don't see any other members...